lib/elixir_analyzer/exercise_test/two_fer.ex

```diff
@@ -132,6 +122,17 @@ defmodule ElixirAnalyzer.ExerciseTest.TwoFer do
     form do
       defp _ignore(_ignore), do: _ignore
     end
+
+    #    # how to match functions named anything _but_ two_fer?
+    #    form do
+    #      def two_fer(_ignore), do: _ignore
+    #      def _ignore(_ignore), do: _ignore
+    #    end
+    #
+    #    form do
+    #      def _ignore(_ignore), do: _ignore
+    #      def two_fer(_ignore), do: _ignore
+    #    end
```

Comment:
I wanted to extend the check about using auxiliary functions to also detect public functions that are not `two_fer/1`, but currently there is no way to match a public function that is called anything but `two_fer`. Do you think it would be possible to add such functionality? Or maybe this could be achieved already by writing two separate features and using `suppress_if` on one of them?

Reply:
I think that was one of my thoughts behind adding the `suppress_if` option, so that it would be easier to write tests like what you are describing

Reply:
I came to the conclusion that it's impractical or even impossible to use `suppress_if` for this purpose. You would need to have a feature and a counter-feature for each possible combination of functions within the module. Also each feature needs to produce a comment, so it's not a good idea to add features only to be used for `suppress_if` of another feature.